core/common/src/test/java/alluxio/security/authorization/AclActionsTest.java

```diff
@@ -11,7 +11,11 @@
 
 package alluxio.security.authorization;
 
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+//import org.junit.Assert;
```

Comment:
This line is no longer necessary.

Reply:
> This line is no longer needed.

Thanks for your advice, I have removed it.

Reply:
But it failed again just because I removed the line?😥